Makefile

```diff
@@ -159,6 +159,9 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 
 .PHONY: deploy
 deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
+	@if [ "$(IMG)" != "controller:latest" ];then \
+		cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG); \
```

Comment:
For me using `make deploy IMG=<>` kustomize automatically updates the controller. Why we need this?

Reply:
strange🤔 I don't see this on my local machine. How is it automatically updating when we didn't add the step to edit the image?

For me, it just deploys Operator with [this](https://github.com/sustainable-computing-io/kepler-operator/blob/afb499daebde69e576f858e5dbe72662bc97dd14/config/manager/kustomization.yaml#L8). Currently, we have `0.5` tag which is incorrect and the pod starts to show `ErrpullImage` that's when I realized we are not changing the image actually.